index.js

```diff
@@ -107,6 +112,30 @@ app.delete('/ideas', (req, res) => {
     res.status(200).send('Ideas successfully deleted.');
 });
 
+//DONATION METHODS
+
+app.get('/donations', (req, res) => {
+    res.status(200).json(ideasDonationSum(req.body.ideaId));
+});
+
+app.get('/donations/sumDifference', (req, res) => {
```

Comment:
What does this API return?

Reply:
It return difference of askedSum and totalDonationSum (for this idea)